openmls/src/lib.rs

```diff
@@ -189,6 +189,7 @@ pub mod storage;
 
 // Private
 mod binary_tree;
+mod skip_validation;
```

Comment:
I suppose we could do that, but in all the places where we optionally disable the check, instead of

```rust
if crate::skip_validation::is_disabled::confirmation_tag() {
  return Err(StageCommitError::ConfirmationTagMismatch)
}
```
we have to do

```rust
#[cfg(feature = "test-utils")]
if crate::skip_validation::is_disabled::confirmation_tag() {
  return Err(StageCommitError::ConfirmationTagMismatch)
}

#[not(cfg(feature = "test-utils"))]
return Err(StageCommitError::ConfirmationTagMismatch)
```

because we don't have access to the `is_disabled` module. That would make the code even more cluttered, which I wanted to avoid.

Reply:
I think I'd be okay with that clutter. Another advantage of hiding this behind a feature flag would be that we could make that `once_cell` dependency optional and for the test-utils feature only.

Reply:
Since once_cell could simply be made an optional dependency I suggest we keep it this way, since that is equivalent and the current version seems simpler.